frontend/src/assets/main.scss

```diff
@@ -124,3 +124,9 @@ td .wrap-block{ width: inherit; }
     text-transform: none;
   }
 }
+
+.p-button {
+  &:hover {
+    opacity: 0.8;
```

Comment:
Is an opacity change what the public slider/toggle button is doing under the hood? If it is, let's do it this way; otherwise I'd recommend doing whatever that component is doing under the hood in lieu of this.

Reply:
<img width="325" alt="Screenshot 2023-05-12 at 8 19 08 AM" src="https://github.com/bcgov/bcbox/assets/117310853/6a1712f4-23b7-4db1-83cf-f7482d084556">

If not wrong, it seems like toggle hover is implemented by hardcoding background colour. Do we want to go this way?